Library/HUMSlider.m

```diff
@@ -472,6 +472,35 @@ - (void)setDesaturatedColor:(UIColor *)desaturatedColor
     self.rightDesaturatedImageView.tintColor = _desaturatedColor;
 }
 
+- (void)setSaturatedColor:(UIColor *)saturatedColor forSide:(HUMSliderSide)side
+{
+    [self imageViewForSide:side saturated:YES].tintColor = saturatedColor;
```

Comment:
My concern here is that you don't have a backing variable for that saturated color for a given side - at this point if the view needs to be redrawn, it will use the same saturated/desaturated color for both sides. 

My thought would be to have a  `setDesaturatedColor:forSide` method for saturated/desaturated and have that call through to a variable something like `leftDesaturatedColor` and `rightDesaturatedColor` - that way you could maintain the independent coloring even if the view has to be reconstructed. 

This would also mean that `setSaturatedColor and `setDesaturatedColor` would probably need to change to call the `setSaturatedColor:forSide` methods, but I don't think that'd be too bad.